x-pack/plugins/security_solution/common/api/entity_analytics/entity_store/entities/common.schema.yaml

```diff
@@ -0,0 +1,154 @@
+openapi: 3.0.0
+info:
+  title: Common Entities Schemas
+  description: Common Entities schemas for the Entity Store
+  version: '1'
+paths: {}
+components:
+  schemas:
+    UserEntityRecord:
```

Comment:
We'll have to keep these schemas in sync with the entity definitions metadata fields, right?
I wonder if there's a way to represent that, maybe deduce the names of the fields from the generated zod records? 

Just pointing it out for the future, I don't necessarily think we need to include that in this PR

Reply:
Yes, unfortunately, we need to keep the API schema and Entity Store schema in sync.
I couldn't figure out a good way to do that automatically. 😞